modules/coreos-layering-configuring-on.adoc

```diff
@@ -0,0 +1,225 @@
+<2> Specifies the Containerfile to configure the custom layered image.
+<3> Specifies the name of the image builder to use. This must be `PodImageBuilder`.
+<4> Specifies the name of the pull secret that the MCO needs to pull the base operating system image from the registry.
+<5> Specifies the image registry to push the newly-built custom layered image to. This can be any registry that your cluster has access to. This example uses the internal {product-title} registry.
+<6> Specifies the name of the push secret that the MCO needs to push the newly-built custom layered image to that registry.
+<7> Specifies the secret required by the image registry that the nodes need to pull the newly-built custom layered image. This should be a different secret than the one used to push the image to your repository.
+
+.. Create the `MachineOSConfig` object:
++
+[source,terminal]
+----
+$ oc create -f <file_name>.yaml
+----
++
+The build process creates a `MachineOSBuilds` object and two new pods.
++
+[source,terminal]
+----
+NAME                                                              READY   STATUS    RESTARTS   AGE
+build-rendered-layered-ad5a3cad36303c363cf458ab0524e7c0           2/2     Running   0          2m40s # <1>
+# ...
+machine-os-builder-6fb66cfb99-zcpvq                               1/1     Running   0          2m42s # <2>
+----
+<1> This is the build pod where the custom layered image is building.
+<2> This pod can be used for troubleshooting.
+
+. If necessary, when the `MachineOSBuild` object has been created and is in the `READY` state, modify the node spec for the nodes where you want to use the new custom layered image:
++
+.. Check that the `MachineOSBuild` object is `READY`. When the `SUCCEEDED` value is `True`, the build is complete.
++
+[source,terminal]
+----
+oc get machineosbuild
+----
++
+.Example output showing that the `MachineOSBuild` object is ready
++
+[source,terminal]
+----
+NAME                                                                PREPARED   BUILDING   SUCCEEDED   INTERRUPTED   FAILED
+layered-rendered-layered-ad5a3cad36303c363cf458ab0524e7c0-builder   False      False      True        False         False
+----
+
+.. Edit the nodes where you want to deploy the custom layered image by adding a label for the machine config pool you specified in the `MachineOSConfig` object:
++
+[source,terminal]
+----
+$ oc label node <node-name> 'node-role.kubernetes.io/<mcp_name>='
+----
++
+--
+where:
+
+node-role.kubernetes.io/<mcp_name>=:: Specifies a node selector that identifies the nodes to deploy the custom layered image. 
+--
++
+When you save the changes, the MCO drains, cordons, and reboots the nodes. After the reboot, the node will be using the new custom layered image.
+
+.Verification
+
+. Verify that the `MachineOSConfig` object contains a reference to the new custom layered image:
++
+[source,terminal]
+----
+$ oc describe MachineOSConfig <object_name>
```

Comment:
An earlier `oc` command used lowercase for `machineosbuild`. Maybe match that style for commands here with `machineosconfig`?
```suggestion
$ oc describe machineosconfig <object_name>
```
Alternatively, flip the earlier `machineosbuild` to `MachineOSBuild`; just one way or the other so that they're consistent in the example commands throughout the PR.